tests/manual-test-cases/Group19-ROBO/19-2-ROBO-Container-Limit.md

```diff
@@ -0,0 +1,50 @@
+Test 19-2 - ROBO - Container VM Limit
+=======
+
+# Purpose:
+To verify that the total container VM limit feature works as expected in a vSphere ROBO Advanced environment.
```

Comment:
This configuration option should not be unique to ROBO - I suggest the main body of this test be in the CI test buckets and this test, in the robo specific setting, uses it in the same way the previous robo test references the regression tests.

Reply:
I brought this up with @mhagen-vmware since the container VM limit feature doesn't apply just to ROBO, but to `vic-machine` in general. This pull request is intended only for ROBO-focused test plans. A CI test plan for this would come when working on https://github.com/vmware/vic/issues/6529 for example.

Reply:
I've updated #6529's acceptance criteria.

Reply:
I'm not sure I understand this response... but I'm not going to push it at this time. We'll simply end up committing the tests directly into the CI test suites and refactoring this test runbook to reference those.